ec_grasp_planner/src/ClashHandRecipes.py

```diff
@@ -1,4 +1,4 @@
-import tf_conversions.posemath as pm
+#import tf_conversions.posemath as pm
```

Comment:
remove the line or add a comment why it is commented

Reply:
ok